src/main/java/com/efficio/fieldbook/web/common/controller/SampleListController.java

```diff
@@ -75,7 +78,11 @@ private void processSampleList(final Integer listId, final HttpServletRequest re
 			final String subObservationVariableName = this.sampleListService.getObservationVariableName(listId);
 			model.addAttribute(SampleListController.SAMPLE_LIST, sampleDetailsDTOs);
 			model.addAttribute(SampleListController.TOTAL_NUMBER_OF_GERMPLASMS, sampleDetailsDTOs.size());
-			model.addAttribute(SampleListController.TABLE_HEADER_LIST, this.getSampleListTableHeaders(subObservationVariableName));
+
+			final boolean isSubobservation = !StringUtils.equals(PLOT_NO_VAR_NAME, subObservationVariableName);
```

Comment:
You can use TermId.PLOT_NO.name(), instead of creating a new constant for PLOT_NO